### PR TITLE
Vote to make a community private or public

### PR DESCRIPTION
**Pull request by cheny0**
- First draft
- Final draft
- Minor fix

### CODE REVIEW

**Comment by cheny0** [comment]
Hi Patryk @osmaczko, this document introduces the procedure of voting to add / remove a community from the directory.

File: docs/en/status-communities/vote-to-make-a-community-private-or-public.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Skip this step if you initiated the voting process. The voting period lasts for 14 days.
+
+1. From the top navigation, select **Votes** to view ongoing votes.
+1. Find the community you want to vote for. If it's a vote for adding a community, select **Don't add** or **Add**. If it's a vote for removing a community, select **Keep** or **Remove**.
+1. Choose the amount of SNT to stake.
+1. If it's a vote for adding a community, select **Vote to not add community** or **Vote to add community**. If it's a vote for removing a community, select **Vote to keep community** or **Vote to remove community**.
+
+## Step 4: Verify the vote {: #verify-vote}
+
+After voting, the 7-day verification period begins. During this period, valid votes are identified.
```
Comment (osmaczko) [inline docs/en/status-communities/vote-to-make-a-community-private-or-public.md:83]: We should emphasize this step is optional, but the more peers participate, the better it is for vote integrity.

File: docs/en/status-communities/vote-to-make-a-community-private-or-public.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+### Initiate a vote for removing a community from Discover
+
+1. From the top navigation, select **Directory** to view all existing communities.
+1. Find the community you want to remove and select :desktop-delete: **Delete** next to the community name.
+1. Choose the amount of SNT you want to stake for voting and select **Confirm vote to remove community**.
+1. Confirm the transaction and pay the gas fee from your wallet.
+
+## Step 3: Cast your vote {: #cast-vote}
+
+Skip this step if you initiated the voting process. The voting period lasts for 14 days.
```
Comment (osmaczko) [inline docs/en/status-communities/vote-to-make-a-community-private-or-public.md:74]: Would be great to add a point about signing a vote.

Comment (cheny0) [reply docs/en/status-communities/vote-to-make-a-community-private-or-public.md]: > Would be great to add a point about signing a vote.

I have applied your comments. Thanks!

**Comment by cheny0** [comment]
@jorge-campo, the article for switching networks using Status Wallet doesn't exist yet. When it's ready I'll add a link to this document and #698.

**Review by jorge-campo — CHANGES_REQUESTED** [review]
Hi Chen, thanks for your content.

Because this article is similar to the 'Vote to feature a Status Community' (PR https://github.com/status-im/help.status.im/pull/698) one, I will only add here two comments relevant to this article.

Please, apply my comments in https://github.com/status-im/help.status.im/pull/698#pullrequestreview-1566570590 to this as well.

- I'd start the article by saying, 'Status Communities can be public or private'. Your intro is a bit... abrupt :)
- 'Whether a community is public or private is determined by a vote, open to all SNT holders, not just community members or Status users.' Please, change this sentence using the active voice. For example, 'A voting process determines ... All SNT holders... can participate on...'

Thanks.

**Comment by cheny0** [comment]
> Hi Chen, thanks for your content.
> 
> Because this article is similar to the 'Vote to feature a Status Community' (PR #698) one, I will only add here two comments relevant to this article.
> 
> Please, apply my comments in [#698 (review)](https://github.com/status-im/help.status.im/pull/698#pullrequestreview-1566570590) to this as well.
> 
>     * I'd start the article by saying, 'Status Communities can be public or private'. Your intro is a bit... abrupt :)
> 
>     * 'Whether a community is public or private is determined by a vote, open to all SNT holders, not just community members or Status users.' Please, change this sentence using the active voice. For example, 'A voting process determines ... All SNT holders... can participate on...'
> 
> 
> Thanks.

Hi @jorge-campo, I have applied your comments as we discussed over the call. Thanks!

**Pull request closed without merging**